CHANGELOG.md

```diff
@@ -1,60 +1,106 @@
+## 2.2.0 (March XX, 2022)
+
+NOTES:
+
+* resource/local_file: Argument `sensitive_content` is `Deprecated`; please use the new resource `local_sensitive_file`.
+
+FEATURES:
+
+* **New data-source/local_sensitive_file**: Similar to the data source `local_file`, but `content` and `content_base64` attributes
+  are marked as _sensitive_.
+  ([#101](https://github.com/hashicorp/terraform-provider-local/pull/101) and
+  [#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
+* **New resource/local_sensitive_file**: Similar to the resource `local_file`, but `content` and `content_base64` arguments
+  are marked as _sensitive_.
+  ([#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
```

Comment:
Per [changelog documentation](https://www.terraform.io/plugin/sdkv2/best-practices/versioning#example-changelog):

```suggestion
* **New Data Source:** `local_sensitive_file` ([#101](https://github.com/hashicorp/terraform-provider-local/pull/101) and [#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
* **New Resource:** `local_sensitive_file** ([#106](https://github.com/hashicorp/terraform-provider-local/pull/106))
```

Refer also to [another provider changelog](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md). The additional notes can optionally be added to the notes section.

Reply:
Sounds good, and consistency is great.

Will add a mention of what those properties provide in the `NOTES:`